warehouse/server/apihandler/handler.go

```diff
@@ -0,0 +1,17 @@
+package apihandler
+
+import (
+	"net/http"
+
+	"github.com/concourse/dutyfree/persistence"
+)
+
+type Handler struct {
+	p persistence.Persistence
+}
+
+func NewApiHandler(p persistence.Persistence) http.Handler {
+	mux := http.NewServeMux()
+	mux.Handle("/api/v1/resources", resourcesHandler{p: p})
```

Comment:
just a side note for my own reference wrt golang: https://github.com/golang/go/wiki/CodeReviewComments#variable-names !